GameData/ROEngines/PartConfigs/RL10A4-2_BDB.cfg

```diff
@@ -174,15 +174,20 @@ PART
 	@title = RL10A-4-2 Series Vacuum Engine
 	@MODULE[ModuleEngineConfigs] 
 	{
-		@configuration = RL10A-4-2
+		@configuration = RL10A-4-1-2
+		@CONFIG
+		{
+			@description = Reworked turbopumps and upgraded for usage in a single engine configuration on Atlas V
+			@minThrust = 97.9
+			@maxThrust = 97.9
+			@massMult = 1.006
+		}
```

Comment:
Not what I meant.  And don't think it makes sense to specify values here.  The engine config should exist in RO core.
I meant that the patch structure that deletes a lot of individual configs, by name, is very brittle.  Especially if you only intend one or a couple configs to be available.  In that case, use a structure like:
```
!CONFIG:HAS[~name[RL10A-4*]] {}
```

Reply:
this is the easiest way to config the RL10-4-2

Reply:
this way i don't have to deprecate the RL10-4-1-2 Config

Reply:
Sure.  But the way that I'm going to merge it is the config keeps its home in RO core.  And I'm suggesting a less brittle way to patch these parts to have only a very limited set of the engine configs available.

Reply:
No, my issue with deprecating is you are changing a config that has existed in RO core, so parts besides this mod (including just RO core!) could have already used that config.  So removing a config globally causes problems.  So if we want to go down that path, we have to leave the config available in RO core, mark that it may go away, and then when you're making new parts like this, they don't include/will delete that deprecated config.  Even then, you shouldn't remove it from any existing parts as that will break any vessels in flight, or craft files that used it.